### PR TITLE
OSDOCS-847 document custom build strategy change for node pull secrets

### PR DESCRIPTION
**Pull request by bmcelvee**
Epic: https://issues.redhat.com/browse/OSDOCS-847

### CODE REVIEW

**Comment by bmcelvee** [comment]
At this stage this PR only has commented-out notes highlighting assemblies and modules that will likely require some sort of modification. These notes will be removed before merging.

Researching to figure out where in the process the node pull secret is logged in `/var/lib/kubelet/config.json`.

**Comment by bmcelvee** [comment]
Since the only change that might impact the user is related to custom builds, I added a statement and cleaned up the module.

**Comment by bmcelvee** [comment]
@ricardomaraschini thanks for the reminder in scrum! Would you mind giving this a review when you get a chance?

**Review by ricardomaraschini — APPROVED** [review]
/lgtm

**Comment by bmcelvee** [comment]
@wzheng1 would you mind giving this PR a review? Thanks!

**Comment by wewang58** [comment]
@bmcelvee  I can not find doc content about https://issues.redhat.com/browse/DEVEXP-423, the pr is just format something, do you guide to where to find Doc related to DEVEXP-423, thanks

**Comment by bmcelvee** [comment]
Hi @wewang58 - I'm sorry this one was a bit confusing to me too. It is my understanding that there are minimal changes in DEVEXP-423 that the user will interact with, only a slight change that might affect users working with the custom build strategy. So this PR is just adding the following note to the custom build strategy documentation:

"If a pull secret for the registry exists in both the namespace and the node, builds default to using the pull secret on the namespace."

https://osdocs-847-node-pull-secrets--ocpdocs.netlify.app/openshift-enterprise/latest/builds/creating-build-inputs.html#builds-custom-strategy_creating-build-inputs

Do you think further documentation is required? If so, I would be happy to add it. Thank you!

**Comment by wewang58** [comment]
@bmcelvee If customers need further document required in future, we will add more, now it's ok.
/lgtm

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions, but this is looking good.

File: modules/builds-custom-strategy.adoc [hunk 1]
```diff
@@ -8,32 +8,14 @@
...
-There is no technical difference between existing strategy secrets and the input
-secrets. However, your builder image might distinguish between them and use them
-differently, based on your build use case.
-
-The input secrets are always mounted into the
-*_/var/run/secrets/openshift.io/build_* directory or your builder can parse the
-`$BUILD` environment variable, which includes the full build object.
+There is no technical difference between existing strategy secrets and the input secrets. However, your builder image might distinguish between them and use them differently, based on your build use case.
 
+The input secrets are always mounted into the `/var/run/secrets/openshift.io/build` directory or your builder can parse the `$BUILD` environment variable, which includes the full build object.
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/builds-custom-strategy.adoc:16]: Thank you! It's my understanding that you need to either reference that directory or invoke the $BUILD variable.

**Pull request merged**